ovos_utils/system.py

```diff
@@ -24,6 +24,72 @@ class MycroftRootLocations(str, Enum):
 
 _USER_DEFINED_ROOT = None
 
+# system utils
+@deprecated("DEPRECATED: use ovos-PHAL-plugin-system", "0.1.0")
+def ntp_sync():
+    # """
+    # Force the system clock to synchronize with internet time servers
+    # """
+    # subprocess.call('service ntp stop', shell=True)
+    # subprocess.call('ntpd -gq', shell=True)
+    # subprocess.call('service ntp start', shell=True)
+    self.bus.emit(Message("system.ntp.sync"))
```

Comment:
self.bus does not exist, you need to create a new connection from scratch. maybe add bus=none to the arguments and then bus = bus or get_mycroft_bus()